user-guide/Standard_Apps/PA/Getting_Started_with_PA/Creating_Activities/PA_Creating_script_tasks.md

```diff
@@ -141,6 +141,9 @@ uid: PA_Creating_script_tasks
       > [!NOTE]
       > In the C# block, never try to directly retrieve data from the DOM instance, as this can have unexpected results.
 
+      > [!IMPORTANT]
+      > Check that the script has three parameters: *Info*, *ProcessInfo* and *ProfileInstance*.
```

Comment:
Are these parameters that the user still needs to add, or is this supposed to have happened sometime during the procedure above? If the user needs to add them, it might be better to change the creation of the parameters into a step of its own in the procedure.

Reply:
At line 84 it mentions to duplicate a template script, so the parameters should already be there. On occasions where people would transform a regular script it might be interesting to inform they're really required.